backend/utils/validation.js

```diff
@@ -4,17 +4,17 @@ const { validationResult } = require('express-validator');
 // (to customize, see express-validator's documentation)
 const handleValidationErrors = (req, _res, next) => {
   const validationErrors = validationResult(req);
-
+  console.log(validationErrors)
```

Comment:
You probably dont want this console.log in production, particularly since you are already sending the info in the response for the error.